File: KarateKlub/KarateKlub.Api/Models/ClanarineNeizmireneClanarineClanaPregledVM.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace KarateKlub.Api.Models
{

    public class ClanarineNeizmireneClanarineClanaPregledVM
    {
        public class Row
        {
            public int id { get; set; }
            public string naziv { get; set; }
            public DateTime datumOd { get; set; }
            public DateTime datumDo { get; set; }
        }
        public List<Row> rows { get; set; }
    }
}

File: Android aplikacija/KarateKlub/app/src/main/java/com/example/x/karateklub/area_clan/model/ClanarineNeizmireneClanarineClanaPregledVM.java
package com.example.x.karateklub.area_clan.model;

import java.io.Serializable;
import java.util.Date;
import java.util.List;

public class ClanarineNeizmireneClanarineClanaPregledVM implements Serializable {
    public static class Row implements Serializable{
        public int id;
        public String naziv;
        public Date datumOd;
        public Date datumDo;
    }

        public List<Row> rows;
}
